src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV, if you'll implement clearing of state on SESSION_DESTROY, then you never need to 'reload' your LOCs list, you need only to initialize it. So 'ready' in your abstracted state means, that your state is ready for complete work, because the most of the cases will be impossible if list of your entities isn't loaded yet.
This 'ready' prop should be implemented with the same way (for convenience and quick perception) in all such functionality when we want to load something only one time per one session.